VikiGolieusova/src/ContactList/Contact.java

```diff
@@ -0,0 +1,27 @@
+package ContactList;
+
+
+public class Contact {
+
+    private String name;
+    private String phone;
+
+    public String getName() {
+        return name;
+    }
+
+    public String getPhone() {
+        return phone;
+    }
+
+    public Contact(String name, String phone){
```

Comment:
try to order blocks by next rules:
1) fields
2) constructor
3) your methods
4) get - set